src/osemgrep/cli_scan/Rule_filtering.ml

```diff
@@ -32,13 +32,11 @@ let get_rule_product_from_metadata (rule : Rule.t) =
   | Some json -> (
       let product_field = JSON.member "product" json in
       let sca_field = JSON.member "sca-kind" json in
-      let interfile = JSON.member "interfile" json in
-      match (product_field, sca_field, interfile) with
-      | _, _, Some (Bool true) -> Interfile
```

Comment:
cc @akuhlens who added this code recently. Not sure if it's ok to remove the Interfile case.

Reply:
no it was me who added this code, interfile was added so I could disable it for the LSP, but I realized interfile rules are still valid in non interfile mode